enterprise/com/src/main/java/org/neo4j/com/Server.java

```diff
@@ -200,17 +202,27 @@ public void start() throws Throwable
             silentChannelExecutor.shutdownNow();
             throw new IOException( ex );
         }
+
+        successfullyStarted = true;
     }
 
     @Override
     public void stop() throws Throwable
     {
         String name = getClass().getSimpleName();
-        msgLog.info( name + " communication server shutting down and unbinding from  " + socketAddress );
+        if ( successfullyStarted )
```

Comment:
To my knowledge LifeSupport should not call stop unless start was successful, so this behaviour should already be in place.